src/mocks/data/draft_reports_object.json

```diff
@@ -0,0 +1,50 @@
+{
+  "status": "processing", // enum: one of ["processing", "failed", "delivered"]
+  "current_stage": "dex-upload",
```

Comment:
current_ seems redundant

Reply:
First off, I want to clarify, this JSON is technically representing the processing status `submission-details` api call. So the top level keys are not actually the report. Only the items inside the `reports` array are the report object structure we are discussing. 

With that in mind, the top level object outside of the `reports` array is high level aspects that the `PS-API` is providing outside of the reports. Many of these high level attributes would be coming directly from the reports, hence the duplication. 

So in this case, `current_stage` and `current_action` is used to represent where the upload/fileSubmission currently is in the pipeline. These would be derived from the reports by PS-API and moved to these high level keys.  Whereas `stage` below inside the report would represent where the item was when that report was created. That's my understanding.